xblock/core.py

```diff
@@ -174,6 +174,15 @@ def validate(self):
         """
         return Validation(self.scope_ids.usage_id)
 
+    def ugettext(self, text):
```

Comment:
Yes, that's the basic idea -- generalizing a common pattern that we're seeing across multiple XBlock projects